test/e2e/framework/util.go

```diff
@@ -503,12 +503,15 @@ func RandomSuffix() string {
 }
 
 // LookForStringInPodExec looks for the given string in the output of a command
```

Comment:
> During review of PR #92127, we found that LookForStringInPodExec function does not have a container name argument. The comments on the function suggested that it intended to have container name as a parameter.

I might have missed  something, but the comments here don't seem to imply that container name was an intentional parameter here initially.

Reply:
I figured it might say "executed in a first pod container" as opposed to "executed in a specific pod container" in the intent was very clear. 

In any case, do you think this change is a good idea? Another alternative is to add a new function [LookForStringInPodExecToContainer](https://github.com/kubernetes/kubernetes/pull/92127/files/426d0bbcb13489c50a52dbbf3b5fef24bb2720d6#diff-eb7b79470992813ea1905e96c298b47bR511) from my earlier PR that allows specifying container name as parameter, and have LookForStringInPodExec call it.

Reply:
> Another alternative is to add a new function LookForStringInPodExecToContainer from my earlier PR that allows specifying container name as parameter, and have LookForStringInPodExec call it.

I think this might be better, that way existing callers can still use `LookForStringInPodExec` without passing in an empty container name.

Reply:
I missed the above comment.
I prefer to see a new function `LookForStringInPodExecToContainer()` which allows to specify a new container name parameter and the function is called from the existing `LookForStringInPodExec()`.

Reply:
@andrewsykim @oomichi Sounds good.. I'll push an update shortly, and clarify the comments on the original function.

Reply:
Fixed in commit https://github.com/kubernetes/kubernetes/pull/95055/commits/334cde51bfab31c9dfe812661050b4782a4654f0